lib/pages/chats/chat_model.dart

```diff
@@ -0,0 +1,21 @@
+class GroupmeChat {
+  String threadId;
+  String userId;
+  String name;
+  String avatarUrl;
+  String lastMessage;
+  int timestamp;
+
+  GroupmeChat({this.threadId, this.userId, this.name, this.timestamp, this.lastMessage, this.avatarUrl});
+
+  factory GroupmeChat.fromJson(Map<String, dynamic> json) {
```

Comment:
```suggestion
  // See: https://dev.groupme.com/docs/v3#chats
  factory GroupmeChat.fromJson(Map<String, dynamic> json) {
```

just to make it easier to find the relevant API docs if we have to touch this in the future

Reply:
I am going to refactor this whole goddamn thing, let it be for now

Reply:
I'll do it myself then